client/modules/workspace/src/JobsListing/JobsListing.tsx

```diff
@@ -95,6 +96,15 @@ export const JobsListing: React.FC<Omit<TableProps, 'columns'>> = ({
     markRead: false,
   });
   const { mutate: readNotifications } = useReadNotifications();
+  const navigate = useNavigate();
+  const handleReuseInputs = (job: TTapisJob) => {
```

Comment:
Since this is copied here and in `JobsDetailModal`, is there somewhere we can put this as a util function? Or import from the file where it's first declared?

Reply:
this used useNavigate, so moved it to a shared component instead.